functional-tests/package.json

```diff
@@ -14,23 +15,26 @@
   ],
   "author": "",
   "license": "MIT",
-  "volta": {
-    "node": "14.18.2"
-  },
   "devDependencies": {
+    "@cucumber/cucumber": "^8.10.0",
     "@nice-digital/wdio-cucumber-steps": "^3.0.0",
-    "@wdio/allure-reporter": "^7.33.0",
-    "@wdio/cli": "^7.33.0",
-    "@wdio/cucumber-framework": "^7.33.0",
-    "@wdio/local-runner": "^7.33.0",
-    "@wdio/selenium-standalone-service": "^7.33.0",
-    "@wdio/spec-reporter": "^7.33.0",
+    "@types/node": "^14.18.23",
+    "@wdio/allure-reporter": "^8.0.0",
+    "@wdio/cli": "^8.0.0",
+    "@wdio/cucumber-framework": "^8.0.0",
+    "@wdio/local-runner": "^8.1.3",
+    "@wdio/selenium-standalone-service": "^8.0.0",
```

Comment:
Check with Martin but I don't think you need the selenium standalone service?

Reply:
Resolved - I've removed the deprecated dependency